### PR TITLE
Add resources

### PR DESCRIPTION
**Pull request by LizzySteger**

### CODE REVIEW

**Review by aganpule — APPROVED** [review]
looks good!

**Comment by jeremyjpark** [comment]
I've changed the merge base from master to Iter3Staging. There are a couple of merge conflicts.

File: features/testfiles/test_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello, world.
```
Comment (susannasouv) [inline features/testfiles/test_file.txt:1]: Should we be including test files? @jeremyjpark

File: app/views/past_events/edit.html.erb [hunk 2]
```diff
@@ -1,4 +1,36 @@
 <h2>Edit Event</h2>
+<<<<<<< HEAD
```
Comment (susannasouv) [inline app/views/past_events/edit.html.erb]: needs to be resolved

**Review by aganpule — APPROVED** [review]
lgtm

File: app/views/past_events/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
+<h2>Past Events</h2>
+=======
```
Comment (susannasouv) [inline app/views/past_events/index.html.erb]: resolution needed here

File: app/views/past_events/new.html.erb [hunk 4]
```diff
@@ -1,3 +1,15 @@
+<<<<<<< HEAD
```
Comment (susannasouv) [inline app/views/past_events/new.html.erb]: and here

File: db/schema.rb [hunk 5]
```diff
@@ -11,7 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Comment (susannasouv) [inline db/schema.rb]: also here

File: features/support/paths.rb [hunk 6]
```diff
@@ -21,6 +21,11 @@ def path_to(page_name)
       '/about_us'
     when /^the past events page$/
       '/past_events'
+<<<<<<< HEAD
```
Comment (susannasouv) [inline features/support/paths.rb]: here

File: app/models/past_event.rb [hunk 7]
```diff
@@ -1,8 +1,14 @@
+<<<<<<< HEAD
```
Comment (susannasouv) [inline app/models/past_event.rb]: resolution needed here

File: app/controllers/past_events_controller.rb [hunk 8]
```diff
@@ -8,6 +8,24 @@ def new
   end
 
   def create
+<<<<<<< HEAD
```
Comment (susannasouv) [inline app/controllers/past_events_controller.rb]: Use the Iter3Staging one

**Review by susannasouv — APPROVED** [review]

**Pull request merged**